tests/utils.py

```diff
@@ -36,9 +36,8 @@ def make_scope_ids(runtime, usage_key):
         xblock.fields.ScopeIds: A ScopeIds object for the block for usage_key
     """
     block_type = 'fake'
-    def_id = runtime.id_generator.create_definition(block_type)
```

Comment:
Can we use [TestRutime](https://github.com/openedx/XBlock/blob/master/xblock/test/tools.py#L108) from edX here? That's specifically created for Testing Purposes. 

Ideally, It would look like :

```
        from xblock.runtime import DictKeyValueStore, KvsFieldData
        from xblock.test.tools import TestRuntime
        
        
        runtime = TestRuntime(services={'field-data': KvsFieldData(kvs=DictKeyValueStore())})

```

The imports can go on top of the file thought. I was testing different things out and the tests still pass if we use id-generator with the above runtime.